src/Protocol/Activity/ANamespace.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+namespace Friendica\Protocol\Activity;
+
+/**
+ * Activity namespaces constants
+ */
+final class ANamespace
```

Comment:
AaaaaaaaaaaaaaaaAAAaaaaaAaaaaah no.

Please move this to `Friendica\Protocol\ActivityNamespace`